.riot/requirements/1b18826.txt

```diff
@@ -14,7 +14,7 @@ mock==5.1.0
 opentracing==2.4.0
 packaging==23.2
 pluggy==1.2.0
-pytest==7.4.3
+pytest==7.4.4
```

Comment:
nit: Should this be pytest==8.0.0? Is this a stale/outdated file?

Reply:
Nope you're right fixing it now